lib/oli_web/live/delivery/student/utils.ex

```diff
@@ -388,7 +388,7 @@ defmodule OliWeb.Delivery.Student.Utils do
   def container_label_for_scheduling_type([:read_by]), do: "Read by: "
   def container_label_for_scheduling_type(_), do: "Due by: "
 
-  def label_for_scheduling_type(:due_by), do: "Due by: "
+  def label_for_scheduling_type(type) when type in [:due_by, "Not yet scheduled"], do: "Due by: "
```

Comment:
I find it a bit odd that this expects a string like 'Not yet scheduled', given that (based on the function name) it should expect a scheduling type, which is an atom.